packages/ecc-client-ga4gh-tes/src/components/tes-create-run/test/tesCreateRun.unit.test.ts

```diff
@@ -1,9 +1,11 @@
-import { html, fixture, expect, elementUpdated } from '@open-wc/testing';
+import { html, fixture, expect } from '@open-wc/testing';
 import { DesignSystem } from '@microsoft/fast-foundation';
 import TESCreateRun from '../definition/tesCreateRun.js';
 
 DesignSystem.getOrCreate().register(TESCreateRun);
 
+// TODO: Add alternative for elementUpdated whereever needed for FAST components. Maybe do it after Lit migration.
```

Comment:
Hi @JaeAeich, 

I reduced some tests because they were failing due to an issue with `elementUpdated` and `FAST`. Since we are now using `tsup` instead of `tsc`, there was no point in keeping `tsc` just for the tests, so I removed it. Now, the tests are run directly from the .ts files without any build. The tests use `@web/dev-server-esbuild`, which is provided by `wtr` itself, for internal compilation to js. I am not sure what the bug was, so I commented out the code for the test where `elementUpdated` was used. I hope this is okay with you. Please let me know if it is not. In any case, we will be migrating to `lit` soon, and it will work perfectly fine with that (I saw in the docs that it supports `lit` and `Stencil`, but not `FAST`, unfortunately).

Reply:
i think this is fine, I'm currently in the process of reworking all the tests anyways.